fighters/common/src/function_hooks/collision.rs

```diff
@@ -98,10 +98,31 @@ unsafe fn get_touch_flag_hook(boma: &mut BattleObjectModuleAccessor) -> i32 {
     original!()(boma)
 }
 
+#[skyline::hook(offset = 0x6ca950)]
+unsafe fn ground_module_update_hook(ground_module: u64) {
+    call_original!(ground_module);
+    let boma = *((ground_module + 0x20) as *mut *mut BattleObjectModuleAccessor);
+    let line = *((ground_module + 0x28) as *mut *mut f32);
+    let shift = VarModule::get_float((*boma).object(), vars::common::instance::ECB_Y_OFFSETS);
```

Comment:
This VarModule call should be within the `is_fighter` if block
```
if (*boma).is_fighter() {
  let shift = ...;
  if !(*boma).is_status_one_of... {
  }
}
```

Reply:
Any reason? Will calling VarModule for a non-fighter boma cause potential issues?

Reply:
well if something is passed into this and it *doesn't* have a VarModule, the game will panic, so it's really just to make sure...

Reply:
fwiw, i've been playing w this for weeks now, and haven't gotten a panic. But for sanity's sake i can change it, sure